### DISCUSSION TITLE
Swapping Chrome license to Firefox due to Manifest V2 being deprecated next month?

### DISCUSSION THREAD

**Discussion by StickyChannel92** [aclap-dev/video-downloadhelper#522] [License & Purchase requests]
### What's your issue?

I have a Chrome license and I would like to know if it is possible to switch to a Firefox one for free or at least a discount, since Chrome is switching to Manifest V3 and Manifest V2 is being deprecated. I want to do this because adblockers would have to overcome this hurdle, and I want to migrate to Firefox or any Firefox-based browser early.

Most of my extensions including uBlock Origin are Manifest V2.

### The 10 first digits (only 10) of your license number or email address

a4f01e7075

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
Sorry, there are no discounts of this type.

**Reply by Hepworks** [reply]
I'd also like to swap my license.  My primary use is you tube downloads for research, and it wasn't obvious that that was impossible until after I'd bought the license.

**Reply by jcfrog** [reply]
please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/I-purchased-the-wrong-premium
